inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/project/RecommenderEditorPanel.java

```diff
@@ -138,7 +138,7 @@ public RecommenderEditorPanel(String aId, IModel<Project> aProject,
         form.add(nameField);
         
         autoGenerateNameCheckBox = new CheckBox(MID_AUTO_GENERATED_NAME,
-                PropertyModel.of(this, "autoGenerateName"));
+                PropertyModel.of(this, MID_AUTO_GENERATED_NAME));
```

Comment:
The MID is the identifier that binds the Wicket CheckBox to its HTML element defined in the respective `.html` file. However, the`autoGenerateName` literal used in the `PropertyModel` refers to the name of a field defined in the RecommenderEditorPanel (`private boolean autoGenerateName;`). The HTML element ID/markup ID  be changed independently from the field name. Java 8 introduced method references, but unfortunately (afaik), there are no field references (yet). So the PropertyModel works using reflection. I think using the MID here in place of the field name is not sensible.